defold-rive/src/res_rive_data.cpp

```diff
@@ -40,13 +40,7 @@ namespace dmRive
 
         //dmRive::DebugBoneHierarchy(&scene_data->m_Roots);
 
-        bool bones_ok = dmRive::ValidateBoneNames(&scene_data->m_Bones);
```

Comment:
Should this be communicated in the editor as well?

Reply:
Hmm, yeah, maybe?